demo/app/src/context/appContext.js

```diff
@@ -5,33 +5,39 @@ const AppContext = createContext();
 
 export default function AppProvider({ children }) {
   const today = new Date();
-  const [geoJson, setGeoJson] = useState();
-  const [bbox, setBbox] = useState();
-  const [dateRange, setDateRange] = useState([
+  /*
+    userParams interface:
+    {
+      bbox: tuple<number>,
+      dateRange: tuple<Date>
+    }
+  */
```

Comment:
suggesting jsdoc here

```suggestion
  /**
   * @param {object} param0
   * @param {[Date, Date]} param0.dateRange
    * @param {number[]} param0.bbox
  */
```

Reply:
I couldn't figure out how to use github to comment on multiple lines :sadpanda: